engines/bops_config/config/routes.rb

```diff
@@ -23,6 +23,8 @@
     end
   end
 
+  resources :legislation, except: %i[show]
```

Comment:
should this route be plural like that `resources :legislations, except: %i[show]`?

Reply:
Decided to use just legislation

https://github.com/unboxed/bops/blob/main/config/initializers/inflections.rb from https://github.com/unboxed/bops/commit/ec075bf350302dd0925bd848bdf62b2cfb6fcc50